views/jobRoles.njk

```diff
@@ -29,6 +46,40 @@
                         </a>
                     {% endfor %}
                 </div>
+
+                <div class="pagination d-flex flex-row justify-content-center">
+                    <div class="d-flex flex-row gap-2 align-items-center mx-3">
+                        <nav>
+                            <ul class="pagination">
+                                {% for i in range(1,pagination.pages+1) %}
+                                    <li class="paginationCard {% if i == pagination.page %}active{% endif %}">
+                                        <a class="card p-3 m-1 " href="?page={{ i }}&limit={{ pagination.limit }}">{{ i }}</a>
+                                    </li>
+                                {% endfor %}
+                            </ul>
+                        </nav>
+                    </div>
+                </div>
+
+                <nav class = "d-flex flex-row justify-content-center mb-5 pb-5">
+                    <ul class="pagination">
+                        {% if pagination.page > 1 %}
+                            <li class="page-item">
+                                <a class="page-link" href="?page={{ pagination.page - 1 }}&limit={{ pagination.limit }}">Previous</a>
+                            </li>
+                        {% endif %}
```

Comment:
As per the ticket the UI looks odd when you have one card at the end of the page but still have content on another page. There are two ways to fix this. Either you can have 5 cards, but risk smaller text, or 1 card per row. Pick one.

Reply:
Overruled by you guys